src/Nethermind/Nethermind.Optimism/OptimismCostHelper.cs

```diff
@@ -15,6 +15,10 @@ namespace Nethermind.Optimism;
 
 public class OptimismCostHelper(IOptimismSpecHelper opSpecHelper, Address l1BlockAddr) : ICostHelper
 {
+    public OptimismCostHelper(IOptimismSpecHelper opSpecHelper, OptimismChainSpecEngineParameters chainSpecEngineParameters) : this(opSpecHelper, chainSpecEngineParameters.L1BlockAddress!)
```

Comment:
Do we have some annotation to indicate that this constructor is used by DI? Otherwise it can be eventually thought of as dead code (due to no usages).

Reply:
We don't actually.